jwst/datamodels/util.py

```diff
@@ -444,3 +445,30 @@ def create_history_entry(description, software=None):
     if software is not None:
         entry['software'] = software
     return entry
+
+
+def get_envar_as_boolean(name, default=False):
```

Comment:
I realize this is existing code that was just copied to a new module, but I would whitelist the falsey values here and raise `ValueError` for an unrecognized string.  This would prevent the code from silently treating strings like "true " (note trailing space) as `False`.